docs: Add Process "How to link an Issue to a PR" (#73)
Pradumnasaraf/open-source-with-pradumna: pages/How-to/guide/linking-issue-with-pr.md

# How to Link and Issue no with a PR ❓
### Things to keep in 🧠
> You can link up to **ten issues** to each pull request.

> The issue and pull request must be in the **same repository**.

## Linking an issue with a PR 🔗 

> 1. On GitHub.com, navigate to the main page of the repository.
> 2. Under your repository name, click **Pull requests**.
> 3. In the list of pull requests, click the pull request that you'd like to link to an issue.

Reviewer: ```suggestion
1. Open the PR in which you want to link the issue. ( Note: If you are a contributor you can only link an issue to your PR. Only maintainers can do with any PR.)
```

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

> 4. In the *right sidebar*, in the "**Development**" section click ⚙️.

Reviewer: ```suggestion
2. In the *right sidebar*, in the "**Development**" section click ⚙️.

```

![image-1](https://user-images.githubusercontent.com/57096457/175424944-f4e98c43-d3b6-497b-911c-45bf5d3eb574.png)

> 8. Click the issue you want to **link to the pull request**.

Reviewer: ```suggestion
3. Click the issue you want to **link to the pull request**.

```

![image-2](https://user-images.githubusercontent.com/57096457/175425148-a4f241c6-e76e-46d7-a47c-a76fcd5b4674.png)

> 9. Wow, **you're successfully** linked the issue with the pull request! 🎉

Reviewer: ```suggestion
4. Wow, **you successfully** linked the issue with the pull request! 🎉

```

# How to Link and Issue no with a PR ❓
### Things to keep in 🧠
> You can link up to **ten issues** to each pull request.

> The issue and pull request must be in the **same repository**.

## Linking an issue with a PR 🔗 
1.  On GitHub.com, navigate to the main page of the repository.
2.  Now [`Create a pull request`](https://opensource.pradumnasaraf.co/#/pages/How-to/guide/creating-pr) first.
3.  And then [`Create an issue`](https://opensource.pradumnasaraf.co/#/pages/How-to/guide/rasing-issue).
4. Under your repository name, click **Pull requests**.
5. In the list of pull requests, click the pull request that you'd like to link to an issue.

> 6. In the *right sidebar*, in the "**Development**" section click ⚙️.

Reviewer: Instead of steps for issue and PR, we can assume the PR and issue is already created

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

Author: Sure, updating the steps.
